setup.py

```diff
@@ -465,13 +483,9 @@ def run(self):
 
         try:
             # Following code is for building the Qualcomm backend.
-            from backends.qualcomm.scripts.download_qnn_sdk import (
-                _download_qnn_sdk,
-                is_linux_x86,
-            )
-
             if is_linux_x86():
                 os.environ["EXECUTORCH_BUILDING_WHEEL"] = "1"
+                from backends.qualcomm.scripts.download_qnn_sdk import _download_qnn_sdk
```

Comment:
Mengwei has this PR https://github.com/pytorch/executorch/pull/15546 maybe can coordinate how to land together or maybe after he merged his change

Reply:
have reverted my changes